src/register_types.cpp

```diff
@@ -14,12 +14,12 @@ using namespace godot;
 static Fmod *_fmod_singleton;
 
 void initialize_fmod_module(ModuleInitializationLevel p_level) {
-	if (p_level != MODULE_INITIALIZATION_LEVEL_SCENE) {
-		 Callbacks::GodotFileRunner::get_singleton();
-
+	if (p_level == MODULE_INITIALIZATION_LEVEL_SCENE)
+	{
+		Callbacks::GodotFileRunner::get_singleton();
 		ClassDB::register_class<Fmod>();
 		_fmod_singleton = memnew(Fmod);
-		Engine::get_singleton()->register_singleton("Fmod", Fmod::get_singleton());
```

Comment:
`Engine::getSingleton()` only seems to be available during the `MODULE_INITIALIZATION_LEVEL_SCENE` stage.

Reply:
That's weird. 
Aren't singleton supposed to be registered as well in previous stages for people wanting a different server for Godot (like physics servers)?
But I guess this stage should be enough for us anyway.

Reply:
Exactly my concern, too. I suppose it is simply a bug (as of Godot 4.alpha14,) or a newly introduced restriction (which seems odd)